src/routes/summary/_SummaryItem.svelte

```diff
@@ -17,6 +18,9 @@
 		class:fill
 		tabindex="0"
 		use:tooltip={!time ? item : undefined}
+		style="font-family: {$font === 'random'
+			? 'Noto Sans JP'
+			: $font}, sans-serif !important; font-weight: 400 !important;"
```

Comment:
I don't really see the point in setting the font family on the summary kana since they're not representative on the font that was quizzed, just a random one